metals/src/main/scala/scala/meta/metals/Linter.scala

```diff
@@ -51,40 +67,63 @@ class Linter(cwd: AbsolutePath) extends LazyLogging {
       )
     )
 
+  private val config: () => Either[String, Option[Input]] =
+    configuration
+      .focus(_.scalafix.confPath)
+      .map[Either[String, Option[Input]]] {
+        case None =>
+          val default = cwd.resolve(Configuration.Scalafix.defaultConfPath)
+          if (Files.isRegularFile(default.toNIO))
```

Comment:
Does this get re-evaluated if the file is created after the warning but the setting remains unchanged? I think we should do the `Files.isRegularFile` check every time so that it's OK to create the file after defining the setting to a non-existent file

Reply:
I think it would make sense to re-run `Files.isRegularFile` every time scalafix is invoked to allow the file to get picked up if it's created after the server starts.